src/lib/s3Uploader.js

```diff
@@ -1,12 +1,26 @@
 import { S3 } from 'aws-sdk';
 
-export const s3 = new S3({
-  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
-  endpoint: process.env.S3_ENDPOINT,
-  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
-  signatureVersion: 'v4',
-  s3ForcePathStyle: true,
-});
+let s3Config;
+
+if (process.env.VCAP_SERVICES) {
+  const bucket = process.env.VCAP_SERVICES.s3[0];
```

Comment:
`VCAP_SERVICES` is a json string in the environment. I know in ruby you need to call `JSON.parse(ENV["VCAP_SERVICES"])` to get the values out. Does node handle that transparently?